Source/Core/Core/HW/Memmap.h

```diff
@@ -111,17 +111,17 @@ double Read_F64(const u32 _Address);
 u32 Read_U8_ZX(const u32 _Address);
 u32 Read_U16_ZX(const u32 _Address);
 
-void Write_U8(const u8 _Data, const u32 _Address);
-void Write_U16(const u16 _Data, const u32 _Address);
-void Write_U32(const u32 _Data, const u32 _Address);
-void Write_U64(const u64 _Data, const u32 _Address);
+void Write_U8(const u8 _var, const u32 _Address);
+void Write_U16(const u16 _var, const u32 _Address);
+void Write_U32(const u32 _var, const u32 _Address);
+void Write_U64(const u64 _var, const u32 _Address);
 
-void Write_U16_Swap(const u16 _Data, const u32 _Address);
-void Write_U32_Swap(const u32 _Data, const u32 _Address);
-void Write_U64_Swap(const u64 _Data, const u32 _Address);
+void Write_U16_Swap(const u16 _var, const u32 _Address);
+void Write_U32_Swap(const u32 _var, const u32 _Address);
+void Write_U64_Swap(const u64 _var, const u32 _Address);
```

Comment:
If you're going to mess with these function signatures anyway, make them sane; _Address is a reserved identifier, and the const is useless.